.travis.yml

```diff
@@ -72,9 +72,37 @@ before_script:
 
 script: .ci/test_script.sh
 
+
 after_success:
   # upload coverage report to coveralls.io
   - if [ "$TEST_TYPE" == "tests" ]; then coveralls; fi
 
 git:
   depth: 3
+
+jobs:
+    include:
+        - stage: deploy
+          if: "branch = master AND tag =~ ^v[0-9]+\\.[0-9]+\\.[0-9]+(a|b|rc)?[0-9]*$"
```

Comment:
Just checking - the single `=` is really the equality operator in this language?

Also, is there a reason you are using `tag` here and not `$TRAVIS_TAG` like below?

Reply:
You know, safety for the win.
`tag` and `branch` are part of this grammar.

https://docs.travis-ci.com/user/conditions-v1